internal/cli/kraft/cloud/instance/stop/stop.go

```diff
@@ -98,6 +98,10 @@ func (opts *StopOptions) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("drain timeout must be at least 1ms")
 	}
 
+	if opts.DrainTimeout > time.Hour {
```

Comment:
Isn't this dictated by the way ukpd is configured?

Reply:
Ah, you are right, I thought it was 1 hour for everyone
![image](https://github.com/unikraft/kraftkit/assets/46280801/65747749-0581-452c-bdc5-f5f47688e7d9)

Reply:
Will remove